test/e2e/providers/exec/docker.go

```diff
@@ -59,3 +59,35 @@ func RunDockerExecCommand(container string, cmd string, workdir string) (
 	// command is successful
 	return 0, string(stdoutBytes), string(stderrBytes), nil
 }
+
+func RunDockerPsCommand(flag string, argument string) (
+	code int, stdout string, stderr string, err error,
+) {
+	args := make([]string, 0)
+	args = append(args, "ps", "--" + flag, argument)
```

Comment:
you can just use a slice literal here: `args := []string{"ps", "--", flag, argument}`.